components/emoji_picker/emoji_picker_overlay.jsx

```diff
@@ -5,7 +5,7 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import {Overlay} from 'react-bootstrap';
 
-import EmojiPicker from './emoji_picker.jsx';
+import EmojiPicker from './';
```

Comment:
This import state looks really weird to me, but it makes sense without moving a bunch of files around

Reply:
Yeah it weirded me out a bit but I didn't want to have to go rearrange a bunch of files